packages/devtools_app/lib/src/screens/memory/primitives/README.md

```diff
@@ -0,0 +1 @@
+This folder contains code that does not depend on other code in the folder `memory`.
```

Comment:
What is the value added by creating a subfolder `memory/primitives`?

Reply:
The value is the same as with `src/primitives`. People, who look on the code, want to understand the code structure, , i.e which code is high level and which is low level, faster. 
The folder `primitives` contains low level code, used by other code. It helps to understand the relations between the libraries.

Reply:
I also plan to create the folder `areas`, with subfolders `chart`, `analysis` and `allocations`, similar to the folder 'screen' in the root.
It will help to navigate the code, especially knowing that we will add new functionality and number of files will increase.

Reply:
sounds good